src/NzbDrone.Core/Download/DownloadService.cs

```diff
@@ -71,7 +72,15 @@ private void DownloadReport(RemoteEpisode remoteEpisode, IDownloadClient downloa
 
             // Get the seed configuration for this release.
             remoteEpisode.SeedConfiguration = _seedConfigProvider.GetSeedConfiguration(remoteEpisode);
-            var indexer = _indexerFactory.GetInstance(_indexerFactory.Get(remoteEpisode.Release.IndexerId));
+
+            IIndexer indexer = null;
```

Comment:
If the `IndexerId` is zero, don't look it up, if the indexer is any other value it should be looked up and fail if the ID is invalid. Right now an invalid ID greater than 0 will not fail.